doc/next/_sources/campaign-budgets.rst.txt

```diff
@@ -0,0 +1,31 @@
+.. index:: single: Campaign Budgets
+.. _campaign_budgets_overview:
+
+Campaign Budgets
+================
+
+Each campaign can have 0 or more budgets. Without budgets, the ads in the campaign will not be shown.
```

Comment:
We should stress that budgets are merely limitations on the campaign spending. Once any budget is set, all of the rest of the budget types are assumed to be unlimited. Deleting budgets is not really a way to ensure teh campaign content will not be live (campaign status is for this), this just means the budgets are unlimited. At least that was the conclusion from our discussion.